scripts/audit-member-usernames/main.go

```
@@ -0,0 +1,478 @@
+		"members", constants.KVBucketNameCommitteeMembers,
+		"v1-mappings", "v1-mappings",
+		"v1-objects", "v1-objects",
+	)
+
+	// ── List all committee-member keys ─────────────────────────────────────────
+	slog.InfoContext(ctx, "listing committee-member keys…")
+	lister, err := membersKV.ListKeys(ctx)
+	if err != nil {
+		return fmt.Errorf("list committee-member keys: %w", err)
+	}
+	defer lister.Stop()
```

golangci-lint (2.12.2)

[error] Error return value of `lister.Stop` is not checked

(errcheck)